pysipp/xml/minidom.py

```diff
@@ -0,0 +1,80 @@
+import locale
+import functools
+from xml.dom.minidom import *
+from xml.dom.minidom import _write_data
+
+
+@functools.total_ordering
+class AttributeSorter(object):
+    def __init__(self, obj, *args):
+        self.obj = obj
+
+    def __lt__(self, other):
+        if self.obj in ('request', 'response'):
+            return True
+        elif other.obj in ('request', 'response'):
+            return False
+        return locale.strcoll(self.obj, other.obj) < 0
```

Comment:
Do you really want locale based collation? Shouldn't that just be `self.obj < other.obj` ?